experimental/test/test_CFG_reach_basic.c

```diff
@@ -0,0 +1,59 @@
+#include "LG_internal.h"
+#include <LAGraphX.h>
+
+int main() {
+
+    char *msg = malloc(256);
+    LAGRAPH_TRY(LAGraph_Init(msg));
+
+    GrB_Matrix output;
+
+    GrB_Matrix adj_matrix_a, adj_matrix_b;
+    GrB_Matrix_new(&adj_matrix_a, GrB_BOOL, 4, 4);
+    GrB_Matrix_new(&adj_matrix_b, GrB_BOOL, 4, 4);
+    GRB_TRY(GrB_Matrix_setElement(adj_matrix_a, true, 0, 1));
+    GRB_TRY(GrB_Matrix_setElement(adj_matrix_a, true, 1, 2));
+    GRB_TRY(GrB_Matrix_setElement(adj_matrix_a, true, 2, 0));
+    GRB_TRY(GrB_Matrix_setElement(adj_matrix_b, true, 0, 3));
+    GRB_TRY(GrB_Matrix_setElement(adj_matrix_b, true, 3, 0));
+    GrB_Matrix adj_matrices[2] = {adj_matrix_a, adj_matrix_b};
```

Comment:
I suppose, e need more tests.
1. We can reuse graphs: different queries to one graph
2. We can reuse grammars: same query to different graphs.